suture_simple_test.go

```diff
@@ -5,12 +5,12 @@ import "fmt"
 type Incrementor struct {
 	current int
 	next    chan int
-	stop    chan bool
+	stop    chan struct{}
```

Comment:
For such channels where I never send a value, I typically use the empty struct type as it consumes the least memory, but that is not too relevant in this example.